prbench-models/src/prbench_models/dynamic3d/utils.py

```diff
@@ -1 +1,26 @@
 """Utils for tidybot environments."""
+
+from prbench.envs.dynamic3d.object_types import MujocoObjectType
+from relational_structs import (
+    Object,
+    ObjectCentricState,
+)
+from spatialmath import SE2, UnitQuaternion
```

Comment:
LGTM! One question - do you think `spatialmath` is going to be a useful utility moving forward, and worth keeping as a dependency?